specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json

```diff
@@ -0,0 +1,910 @@
+          "description": "The metric display description."
+        },
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Bytes",
+            "Count",
+            "Milliseconds"
+          ],
+          "x-ms-enum": {
+            "name": "MetricUnit",
+            "values": [
+              {
+                "value": "Bytes"
+              },
+              {
+                "value": "Count"
+              },
+              {
+                "value": "Milliseconds"
+              }
+            ],
+            "modelAsExtensible": true
+          },
+          "description": "The metric unit"
+        },
+        "aggregationType": {
+          "type": "string",
+          "enum": [
+            "Average",
+            "Count",
+            "Total"
+          ],
+          "x-ms-enum": {
+            "name": "MetricAggregationType",
+            "values": [
+              {
+                "value": "Average"
+              },
+              {
+                "value": "Count"
+              },
+              {
+                "value": "Total"
+              }
+            ],
+            "modelAsExtensible": true
+          },
+          "description": "The metric aggregation type"
+        },
+        "dimensions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetricDimension"
+          },
+          "description": "The metric dimensions."
+        }
+      },
+      "type": "object",
+      "required": [
+        "unit",
+        "aggregationType"
+      ],
+      "description": "A metric emitted by service."
+    },
+    "ServiceSpecification": {
+      "properties": {
+        "metricSpecifications": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Metric"
+          },
+          "description": "List of metric specifications."
+        }
+      },
+      "type": "object",
+      "description": "The service metric specifications."
+    },
+    "MetricProperties": {
+      "properties": {
+        "serviceSpecification": {
+          "$ref": "#/definitions/ServiceSpecification",
+          "description": "The service specifications."
+        }
+      },
+      "type": "object",
+      "description": "Metric properties."
+    },
+    "Operation": {
+      "description": "An operation.",	
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The operation name."
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "description": "The operation display name."
+        },
+        "origin": {
+          "type": "string",
+          "description": "Origin of the operation."
+        },
+        "properties": {
+          "$ref": "#/definitions/MetricProperties",
+          "description": "Operation properties format."
+        }
+      },
+      "type": "object",
+      "required": [
+        "name"
+      ]
+    },
```

Comment:
Looking at the api, the `Operation` model seems readonly model where its can be only read through `Operations.list`. If so should we mark all properties of this model as `readonly`? 

The new models representing the new properties of Operation are:

`MetricProperties`
`ServiceSpecification`
`Metric`

should we mark properties of all these models are `readonly`? given they are not used else where.

Reply:
Yes, done.